File: backend/src/main/java/store/andefi/utility/Period.java
package store.andefi.utility;

public enum Period {
  LAST_30_DAYS("last_30_days"),
  LAST_60_DAYS("last_60_days"),
  LAST_90_DAYS("last_90_days");

  private final String value;

  Period(String value) {
    this.value = value;
  }

  @Override
  public String toString() {
    return this.value;
  }
}

File: frontend/app/src/main/java/store/andefi/utility/Period.kt
package store.andefi.utility

enum class Period(private val value: String) {
    LAST_30_DAYS("last_30_days"),
    LAST_60_DAYS("last_60_days"),
    LAST_90_DAYS("last_90_days");

    override fun toString(): String {
        return this.value
    }
}
